addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -1,661 +1,662 @@
-"","","","","","","","account_tax_template_p_iva4_sc","account_tax_template_p_iva4_isp","",""
-"","","","","","","","account_tax_template_p_iva4_bi","account_tax_template_p_iva4_isp_bi","",""
-"","","","","","","","account_tax_template_p_iva10_bc","account_tax_template_p_iva10_isp","",""
-"","","","","","","","account_tax_template_p_iva10_sc","account_tax_template_p_iva10_isp","",""
-"","","","","","","","account_tax_template_p_iva10_bi","account_tax_template_p_iva10_isp_bi","",""
-"","","","","","","","account_tax_template_p_iva21_bc","account_tax_template_p_iva21_isp","",""
-"","","","","","","","account_tax_template_p_iva21_sc","account_tax_template_p_iva21_isp","",""
-"","","","","","","","account_tax_template_p_iva21_bi","account_tax_template_p_iva21_isp_bi","",""
-"","","","","","","","account_tax_template_s_iva4b","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva4s","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva10b","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva10s","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva21b","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva21s","account_tax_template_s_iva0_isp","",""
-"","","","","","","","account_tax_template_s_iva21isp","account_tax_template_s_iva0_isp","",""
-"fp_rnrisp","","Régimen Nacional Revendedor con Inversión del sujeto pasivo","","","","","account_tax_template_s_iva21isp","account_tax_template_s_iva0_isp","",""
-"fp_reagyp_a","","REAGYP - Agricultura","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva12_agr","",""
-"","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_irpf2","",""
-"","","","","","","","account_tax_template_p_iva0_s_bc","account_tax_template_p_iva12_agr","",""
-"","","","","","","","account_tax_template_p_iva0_s_bc","account_tax_template_p_irpf2","",""
-"fp_reagyp_gp","","REAGYP - Ganadería y pesca","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva105_gan","",""
-"","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_irpf2","",""
+"id","sequence","name",,"name@es","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id","account_ids/account_src_id","account_ids/account_dest_id"
+"fp_nacional","1","Domestic",,"Régimen Nacional","1","1","base.es","","","","",""
+"fp_intra_private","2","EU private",,"EU privado","1","","","base.europe","","","",""
+"fp_intra","3","Intra-community",,"Régimen Intracomunitario","1","1","","base.europe","account_tax_template_p_iva0_s_bc","account_tax_template_p_iva0_ic_bc","",""
+"","",,,"","","","","","account_tax_template_p_iva0_s_sc","account_tax_template_p_iva0_ic_sc","",""
```

Comment:
```suggestion
"","","","","","","","","account_tax_template_p_iva0_s_sc","account_tax_template_p_iva0_ic_sc","",""
```
NBD: Could we keep using double quotes to identify empty cells? (consistent style)